dds_cli/__main__.py

```diff
@@ -79,7 +82,8 @@
     "\n[green](  ) ) (  (  )[/]   [bold]SciLifeLab Data Delivery System",
     "\n[green] ︶  (  ) ) ([/]    [blue][link={0}]{0}[/link]".format(dds_cli.__url__),
     f"\n[green]      ︶ (  )[/]    [dim]Version {dds_cli.__version__}",
-    "\n[green]          ︶\n",
+    "\n[green]          ︶",
+    f"\n[green]Current user:[/] [red]{username}",
```

Comment:
Since row 76 might return `None` if something fails in the API call to get the username, I'm thinking we should have the row here as 
```
f"\n[green]Current user:[/] [red]{username}" if username else None
```

Reply:
Otherwise it's going to show up as `Current user: None`

Reply:
Done!